src/sherpa_ai/error_handling/agent_error_handler.py

```diff
@@ -11,14 +11,14 @@ class AgentErrorHandler:
 
     def __init__(self):
         self.error_map = {
-            # openai.APIError: "OpenAI API returned an API Error",
-            # openai.APIConnectionError: "Failed to connect to OpenAI API",
-            # openai.RateLimitError: "OpenAI API request exceeded rate limit",
-            # openai.AuthenticationError: (
-            #     "OpenAI API failed authentication or incorrect token"
-            # ),
-            # openai.Timeout: "OpenAI API Timeout error",
-            # openai.InvalidRequestError: "OpenAI API invalid request error",
+            openai.APIError: "OpenAI API returned an API Error",
```

Comment:
This is not going to work @Eyobyb, this won't resolve the issue we were navigating and if you see this is failing on the test run. And you need to resolve that.